toast-tk-runtime/src/main/java/com/synaptix/toast/runtime/parse/TestParser.java

```diff
@@ -58,9 +58,15 @@ private ITestPage buildTestPage(List<String> lines, String pageName, String file
 	}
 
 	public ITestPage readString(String scenarioAsString, String scenarioName) throws IllegalArgumentException, IOException {
+		// Add error to tests sonar github plugin
 		String[] split = StringUtils.split(scenarioAsString, "\n");
 		ArrayList<String> list = new ArrayList<>(Arrays.asList(split));
 		return buildTestPage(list, scenarioName, null);
 	}
 
+	public void test(){
+		String toto = null;
+		toto.length();
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'toto' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.synaptix-labs.com/coding_rules#rule_key=squid%3AS2259)